ui/src/app/pages/workspace/workspace-nav-bar.tsx

```diff
@@ -142,6 +143,8 @@ const CdrVersion = (props: {
 const tabs = [
   { name: 'Data', link: 'data' },
   { name: 'Analysis', link: 'notebooks' },
+  //  Add this option once we are ready to show the new analysis tab for Apps
```

Comment:
Created a story to keep track of things that needs to be cleaned once the FF is set to true on all environments https://precisionmedicineinitiative.atlassian.net/browse/RW-9130